templates/person/detail.html

```diff
@@ -164,37 +165,113 @@ <h3 id="subordinates"><i class="fa fa-fw fa-share-alt fa-rotate-90"></i> {% tran
+                $('#org-chart-container').flickity({
+                    initialIndex: org_count - 1,
+                    wrapAround: true,
+                });
+            }
+        
+            // Build charts and add year
+            $.each({{command_chain|safe}}, function(index, value) {
+                cellID = "command-chart-" + index
+                cellYearID = "command-chart-year-" + index
 
-            /*
-            $.getJSON(command_url, function(data) {
-                // Print hierarchy
-                console.log(data);
+                obj = JSON.parse(value);
+                node_list = addMultiFont(JSON.parse(obj['nodes']));
+                edge_list = JSON.parse(obj['edges']);
+                when = obj['when'];
 
-                // Load charts library
-                google.charts.load('current', {packages:['orgchart']});
-                google.charts.setOnLoadCallback(drawChart);
+                nodes = new vis.DataSet(node_list);
+                edges = new vis.DataSet(edge_list);
 
-            });
+                container = document.getElementById(cellID);
+                data = {
+                    nodes: nodes,
+                    edges: edges,
+                };
 
-            function drawChart() {
-                var dataTable = new google.visualization.DataTable();
-            }
-            */
+                options = {
+                    nodes:{
+                        shape: 'box',
+                        fixed: {
+                          x:true,
+                          y:true,
+                        },
+                        shapeProperties: {
+                          borderRadius: 2,
+                        },
+                        color: {
+                            background: '#f4f4f4',
+                            border: '#D6D6D6',
+                            highlight: { // Change color of node on click
+                                background: '#EBEBEB',
+                                border: '#D6D6D6',
+                            },
+                        },
+                        font: {
+                            color: '#777777',
+                            face: 'Open Sans',
+                        },
+                    },
+                    layout: {
+                        hierarchical: {
+                          enabled: true,
+                          levelSeparation: 100, // Distance between levels
+                          nodeSpacing: 300, // Distance between nodes
+                          direction: 'DU', // Inverts chart
+                        }
+                    },
+                    interaction: {
+                        zoomView: false,
+                        dragView: false,
+                    },
+                };
+
+                network = new vis.Network(container, data, options);
+
+                $('#' + cellYearID).append(when);
+
+                // Bind click event to individual nodes
```

Comment:
Using JS to mimic link functionality makes me a little bit nervous. This seems to work fine, I just don't trust JS to work as we anticipate across all browsers... any chance we could do something like:

1. add anchor tags around the nodes
2. generate the URLs in the view code
3. update the `href` attribute for each node in this loop

Reply:
Yes, I agree...however, rendering HTML inside nodes is not supported by all browsers: 
https://github.com/almende/vis/blob/develop/examples/network/nodeStyles/HTMLInNodes.html
http://visjs.org/examples/network/nodeStyles/HTMLInNodes.html

So, instead, I used one of the vis.js click events:
https://github.com/almende/vis/blob/develop/examples/network/events/interactionEvents.html#L102

I am going to try and render the *complete* url in python, though!

Reply:
Ahh, gotcha. JS libraries strike again... given the constraints, this is a nice solution!